src/shared/storage/index.js

```diff
@@ -775,7 +775,13 @@ const purge = () =>
  */
 const initialise = (getEncryptionKeyPromise) =>
     getEncryptionKeyPromise().then((encryptionKey) => {
-        realm = new Realm(assign({}, config, { encryptionKey }));
+        // For some reason `getRealm` doesn't get called before this function when running in the Chrome debugger
+        if (__MOBILE__ && /Chrome/.test(navigator.userAgent)) {
+            const Realm = getRealm();
```

Comment:
`initialise` is called on app entry (both desktop & mobile). When Chrome debugging is enabled, `initialise` is called before `Realm` constructor is assigned to the global variable. 

This solution removes the missing constructor error but throws an exception [here](https://github.com/iotaledger/trinity-wallet/blob/develop/src/shared/storage/index.js#L797) because the global constructor remains `undefined`.

Reply:
I suggest to remove the _direct_ global assignment and assign correct constructor to `Realm` inside the initialisation function. This should also work on desktop because of [this](https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/src/index.js#L43).

Reply:
@rajivshah3 

I've made the above [changes](https://github.com/iotaledger/trinity-wallet/commit/78dc7fe94f3984066d43ec0e79fc6e77af5b267f). Lemme know your thoughts on the approach and I'll open a PR against this branch.

Reply:
Looks good! I like that approach better

Reply:
@rajivshah3 https://github.com/iotaledger/trinity-wallet/pull/1177